fulp_modules/features/halloween/2021/shovel_knight_outfits/spectre_knight.dm

```diff
@@ -0,0 +1,30 @@
+/**
+ * Spectre Knight
+ * From: Shovel knight
+ * By: Sheets
+ */
+
+///suit
+
+/obj/item/clothing/suit/costume_2021/spectre_suit
+	name = "Spectre knight's robes"
```

Comment:
I dont know if this is a name, if it isnt, this isnt supposed to start capitalized, same for the others.